libraries/src/Updater/Update.php

```diff
@@ -539,7 +539,9 @@ public function loadFromTuf(TufMetadata $metadataTable, string $url, $minimumSta
     {
         $tufFetcher = new TufFetcher(
             $metadataTable,
-            $url
+            $url,
+            Factory::getDbo(),
```

Comment:
This method is deprecated

```suggestion
            Factory::getContainer()->get(DatabaseDriver::class),
```